### PR TITLE
FINERACT-1375:ClientEntityPersonDataTable

### PR DESCRIPTION
**Pull request by rrpawar96**
## Description
Describe the changes made and why they were made instead of how they were made.

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [ ] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by francisguchie** [comment]
@bharathcgowda @rrpawar96 this has been tested by me LGTM

**Comment by bharathcgowda** [comment]
Hello @luckyman20 @BLasan @spider0061  could you please help in the review of this code, please.

File: app/views/system/createdatatable.html
Comment (luckyman20) [inline app/views/system/createdatatable.html]: Is indentation correct here?

**Review by luckyman20 — APPROVED** [review]
Just a small issue of indentation otherwise LGTM!

**Comment by rrpawar96** [comment]
Yes, I have fixed the Indentation, but let's merge after the backend gets merged. 
https://github.com/apache/fineract/pull/1979

**Comment by bharathcgowda** [comment]
Thank you @luckyman20 for the review.  and @rrpawar96  will merge after the backend is reviewed, thanks.

**Comment by rrpawar96** [comment]
Hi @bharathcgowda, The Backend of this PR in fineract is not yet merged. So If there will be any changes happened in the backend then I will create new PR here in community-app of this feature following this PR which will reflect that changes.

**Pull request merged**
